kbs/src/api_server.rs

```diff
@@ -125,7 +130,7 @@ impl ApiServer {
                         (1024 * 1024 * http_config.payload_request_size) as usize,
                     ))
                     .service(
-                        web::resource([kbs_path!("{base_path}{additional_path:.*}")])
+                        web::resource([kbs_path!("{plugin}{resource_path:.*}")])
```

Comment:
I always find this macro a little confusing. Can you add a comment to describe what is happening (like how is it splitting the path) or maybe it would be clearer to split the path later using more standard rust?